pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -1638,6 +1638,7 @@ void expireMessagesForAllSubscriptions(String topic, long expireTimeInSeconds)
 
     /**
      * Peek messages from a topic subscription.
+     * This method will show server marker, uncommitted and aborted messages for transaction by default.
```

Comment:
This seems inconsistent with the pulsar-client-tool. [The proposal](https://github.com/apache/pulsar/pull/22746/files?short_path=936b339#diff-936b3395e5efb7b110af6294ba1a0a6a597ca6ce612b57542a37ba3956b3d020)  says that we shouldn't show these marker messages by default.

Reply:
This is the original method; a comment has been added here to alert the caller.

Reply:
> [The proposal](https://github.com/apache/pulsar/pull/22746/files?short_path=936b339#diff-936b3395e5efb7b110af6294ba1a0a6a597ca6ce612b57542a37ba3956b3d020) says that we shouldn't show these marker messages by default.

The purpose of pip is to change the behavior of the CLI, not the behavior of this interface.